### PR TITLE
Enable tokens navigate bug

### PR DESCRIPTION
**Pull request by CFDevelopment**
- enable tokens navigation bug fix

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
     "react-native": "0.55.4",
     "react-native-camera": "^1.1.4",
     "react-native-elements": "^0.19.1",
+    "react-native-keyboard-avoid": "^1.2.0",
+    "react-native-keyboard-aware-scroll-view": "^0.6.0",
+    "react-native-keyboard-aware-view": "0.0.14",
     "react-native-linear-gradient": "^2.4.0",
```
Comment (GregTheGreek) [inline package.json]: do you need all three?

File: src/screens/setup/create/CreateWalletName.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from "react";
-import { View, TouchableOpacity, ScrollView, StyleSheet, TextInput, Image, AsyncStorage, Dimensions, Text, Platform } from "react-native";
-import { KeyboardAvoidingView } from 'react-native';
+import { View, TouchableOpacity, TouchableWithoutFeedback, StyleSheet, Image, Text, Platform, Keyboard, Animated, KeyboardAvoidingView, ScrollView     } from "react-native";
+import { KeyboardAwareView } from 'react-native-keyboard-aware-view'
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view'
```
Comment (GregTheGreek) [inline src/screens/setup/create/CreateWalletName.js]: need both?

Comment (ChainSafe) [reply src/screens/setup/create/CreateWalletName.js]: @CFDevelopment

File: src/screens/setup/create/CreateWalletName.js [hunk 3]
```diff
@@ -79,53 +79,54 @@ class CreateWalletName extends Component {
...
+                                customStyles={styles.button}
+                            />
+                        </View>                  
+                        <View style={{ alignItems:'center'}} >    
+                            <View style={{ alignItems:'center'}} >
+                                <Text style={styles.textFooter} >Powered by ChainSafe </Text> 
+                            </View>  
+                        </View>   
+                </View>
+            </TouchableWithoutFeedback>
```
Comment (priom) [inline src/screens/setup/create/CreateWalletName.js]: chris..........

File: src/screens/setup/create/CreateWalletName.js (hunk 2 above)
Comment (CFDevelopment) [reply src/screens/setup/create/CreateWalletName.js]: fixed

**Comment by CFDevelopment** [comment]
- fixed

**Pull request merged**